static/app/components/search/sources/routeSource.tsx

```diff
@@ -2,22 +2,25 @@ import {Component} from 'react';
 import {RouteComponentProps} from 'react-router';
 import flattenDepth from 'lodash/flattenDepth';
 
+import HookStore from 'sentry/stores/hookStore';
 import {Organization, Project} from 'sentry/types';
 import {createFuzzySearch, Fuse} from 'sentry/utils/fuzzySearch';
 import replaceRouterParams from 'sentry/utils/replaceRouterParams';
 import withLatestContext from 'sentry/utils/withLatestContext';
 import accountSettingsNavigation from 'sentry/views/settings/account/navigationConfiguration';
 import organizationSettingsNavigation from 'sentry/views/settings/organization/navigationConfiguration';
 import projectSettingsNavigation from 'sentry/views/settings/project/navigationConfiguration';
-import {NavigationItem} from 'sentry/views/settings/types';
+import {NavigationItem, NavigationSection} from 'sentry/views/settings/types';
 
 import {ChildProps, ResultItem} from './types';
 import {strGetFn} from './utils';
 
+type NavigationSectionsType = Array<NavigationSection>;
 type Config =
   | typeof accountSettingsNavigation
   | typeof organizationSettingsNavigation
-  | typeof projectSettingsNavigation;
+  | typeof projectSettingsNavigation
+  | NavigationSectionsType;
```

Comment:
All settingsNavigation are of time NavigationSectionType. I wonder why we needed to type them one by one. Are there some of them that we don't want to include?

Reply:
@evanpurkhiser I think you might know

Reply:
The return type is right, but not the props each of these functions accept.

It's possible this could be tightened up

Reply:
Got it. These are coming from getsentry so I don't have the exact type but just noticed they're the same type as organizationSettingsNavigation so actually this is not needed. Removing in the next commit.